docs/appendix/machine-specific-install.rst

```diff
@@ -94,9 +95,9 @@ dependencies.
 
 **Step 3: Clone the MELODIES-MONET GitHub repository** ::
 
-    $ git clone git@github.com:NOAA-CSL/MELODIES-MONET.git
+    $ git clone https://github.com/NOAA-CSL/MELODIES-MONET.git
 
-    End step. At the end of working with MELODIES MONET, deactivate the 
+    At the end of working with MELODIES MONET, deactivate the 
     conda environment::
 
     $ conda deactivate
```

Comment:
I think we could just remove this. The Hera section doesn't have it, and it isn't related to "Clone the MELODIES-MONET GitHub repository", which it is nested under.

![image](https://user-images.githubusercontent.com/15079414/157147550-c0eab6ca-f0ba-41ae-971d-d6133654bb37.png)